Character.cs

```diff
@@ -8,16 +8,26 @@ namespace ConsoleInitApp
 {
     class Character
     {
-        public string CharacterName { get; set; }
-        //public Action ActionTaken { get; set; }
-        public int HPValue { get; set; }
-        public int DamageValue { get; set; }
+        public string Name { get; set; }
```

Comment:
This was just a pet-peeve of mine. I like for property names to be as concise as possible. Additional words like Value don't add any useful information. And for `CharacterName` we are in the `Character` class its kind of implied.